crt_portal/cts_forms/templates/forms/widgets/crt_date_entry.html

```diff
@@ -0,0 +1,6 @@
+<div class="usa-hint" id="event-date-start-hint">mm/dd/yyyy</div>
+    <div class="usa-date-picker">
+        <input class="usa-input" id={{ widget.name }} name="event-date-start" type="text" aria-describedby="event-date-start-label event-date-start-hint">
```

Comment:
You probably want `id="{{ widget.name }}"` (note quotes)

Re: `aria-describedby` -- not sure where these ids are? Does this work properly when you use a screen reader?

Reply:
This was with the widget suggested by Jacklyn. @Jacklynn would you know the answer to James' question?

Reply:
`aria-describedby` is documented here, https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute -- just wanted to make sure this attribute value points to valid html ids. Are these ids automatically added by the javascript widget?

Reply:
This implementation has changed considerably. This file is changed and currently not linked because date search fields are made text fields for now. I'm thinking of leaving this in case we come back to enhance. Your thoughts @cantsin ?